src/pages/OwnerPage/Header/HeaderBanners/HeaderBanners.jsx

```diff
@@ -4,40 +4,48 @@ import { useParams } from 'react-router-dom'
 import config from 'config'
 
 import { useOwnerPageData } from 'pages/OwnerPage/hooks'
-import { useAccountDetails } from 'services/account'
+import { useAccountDetails, usePlanData } from 'services/account'
 
 import ExceededUploadsAlert from './ExceededUploadsAlert'
 import GithubConfigBanner from './GithubConfigBanner'
 import ReachingUploadLimit from './ReachingUploadLimit'
 
-const MAX_UPLOADS_NUMBER = 250
-const REACHING_UPLOAD_LIMIT = 225
-
-const useUploadsInfo = () => {
+const useUploadsInfo = (planData) => {
   const { owner } = useParams()
   const { data: ownerData } = useOwnerPageData({ username: owner })
   const numberOfUploads = ownerData?.numberOfUploads
 
-  const isUploadsExceeded = numberOfUploads >= MAX_UPLOADS_NUMBER
-  const isUploadsReachingLimit =
-    numberOfUploads < MAX_UPLOADS_NUMBER &&
-    numberOfUploads >= REACHING_UPLOAD_LIMIT
-
+  // If monthlyUploadLimit is not defined, we consider the account can have an
+  // unlimited amount of uploads
+  const monthlyUploadLimit = planData?.plan?.monthlyUploadLimit
+  const isUploadsExceeded = monthlyUploadLimit
+    ? numberOfUploads >= monthlyUploadLimit
+    : false
+  const isUploadsReachingLimit = monthlyUploadLimit
+    ? !isUploadsExceeded && numberOfUploads >= 0.9 * monthlyUploadLimit
```

Comment:
Ah smart, I like the 90% trick, I think this is a fair assumption and we can tweak it later. In fact, could you ask product how one determines approaching upload limit? It could be a percentage of your total, or a flat value. I prefer percentages but might be worth confirming w/ the law 😂

Reply:
Confirmed with product